ansible/roles/rancher/tasks/rancher.yml

```diff
@@ -8,7 +8,7 @@
     helm install rancher-stable/rancher
       --name rancher
       --namespace cattle-system
-      --version {{ rancher_version }}
+      --version {{ rancher__rancher_version }}
```

Comment:
Why are we introducing the `rancher` stutter here?

Reply:
I was thinking I would re-use the rancher-specific var, but that turned out not to be necessary. I'll just encode the default here.